mmk1.tex

```diff
@@ -786,10 +788,12 @@ \subsection{Theorembeweis mit Resolutionsverfahren}
 Vorgehen: Erweiterung von $\mathcal S $ zu $\mathcal S^* = \left\{S_1 \dots S_n , \neg T \right\}$ Und Resolutionieren bis leere Klausel erzeugt wird. \\
 Erklärung: Statt Beweis wird Unerfüllbarkeit seines Gegenteils gezeigt.
 
-\begin{cookbox}{Tautologie beweisen: }
-	\item Wahrheit auf KNF bringen
-	\item Gegenteil auf KNF bringen
-	\item Zeige, dass Gegenteil \{ \} ist.
+\begin{cookbox}{Tautologie beweisen}
+	% \items were causing a bug here, for some reason can't be used with normal text 
+	Beweis durch Widerspruch: $\neg w \equiv f$: \\
+	1. Negiere Aussage \\
+	2. Bringe Aussage in KNF \\
+	3. Zeige Kontradiktion
```

Comment:
The issue here is that cookbox opens an enumerate environment which expects items. You either put the line without item into the cookbox header _Tautologie beweisen durch Widerspruch ..._ or you apply the following suggestion:

```suggestion	
	\item[] Beweis durch Widerspruch: $\neg w \equiv f$:
	\item Negiere Aussage
	\item Bringe Aussage in KNF
	\item Zeige Kontradiktion
```